index.js

```diff
@@ -29,14 +27,14 @@ function main(fn, fp, data, opts) {
 		});
 	}
 
-	var json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
+	const json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
 
 	return fn(fp, json, {mode: opts.mode});
 }
 
 module.exports = function (fp, data, opts) {
```

Comment:
I stand corrected, cool! But that will still name the function `exports` though?